### PR TITLE
Fix/select component

### PR DESCRIPTION
**Pull request by antkaynak**
### Breaking Changes
- Select component issues fixed -- selected no longer supported
- see related stories for more information
### Changed
- Removed default import from StepProgressBar index **Hot Fix

**Note that this is a breaking change. Select component's selected prop changed to value and value prop changed to valueKey.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.20.3] - 2020-04-27
```
Comment (emredoganm) [inline CHANGELOG.md:7]: This is a breaking change, I think we should make it more higher version. If you remember, we even talked about upgrading it to version 2.0.0. Please double check with @erdoganoksuz and @cihadhoruzoglu for more detailed information. I am not very sure about this.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "quarkify",
-  "version": "1.20.2",
+  "version": "1.20.3",
```
Comment (emredoganm) [inline package.json:3]: version check please

Comment (antkaynak) [reply package.json]: Confirmed to 2.0.0. Waiting for approval :)

**Comment by emredoganm** [comment]
LGTM but we should wait for Erdoğan and Cihad's responses before merging.

**Pull request merged**